### ISSUE TITLE
Update smart contract uploading docs

### ISSUE THREAD

**Issue by ezekiiel** [CosmosContracts/docs#21]
The docs [suggest](https://docs.junochain.com/smart-contracts/tutorial-erc-20/compile#uploading) that running `junod tx wasm store ...` will give some JSON output from which the code ID of the contract can be extracted. Walking through the commands in those docs though the output looks like yaml:

```
gas estimate: 1836558
code: 0
codespace: ""
data: ""
gas_used: "0"
gas_wanted: "0"
height: "0"
info: ""
logs: []
raw_log: '[]'
timestamp: ""
tx: null
txhash: 1405DEEA2C3947526B265FDD199578A7377143FAB397C034A953E1866B06FEF6
```

I can then get the code ID by running `junod query tx <txhash>` and looking at the `raw_log` field.

Is this the expected way to go about this now? If it is I can open a PR to update the docs.

My junod config:
```
$ junod version                                                        
v2.0.0-alpha.1
$ junod config 
{
	"chain-id": "",
	"keyring-backend": "os",
	"output": "text",
	"node": "https://rpc.uni.junomint.com:443/",
	"broadcast-mode": "sync"
}
```

**Comment by ezekiiel** [comment]
Happens with `v2.0.0-alpha.3` as well which seems to be the current version to use.

**Comment by giansalex** [comment]
Also it need to run `junod q tx 1405DEEA2C3947526B265FDD199578A7377143FAB397C034A953E1866B06FEF6` to get logs result

**Comment by ezekiiel** [comment]
Yep. I think the diff for getting the code ID now looks something like this:

```diff
-RES=$(junod tx wasm store cw_erc20.wasm  --from <your-key> --chain-id=<chain-id> --gas auto -y)
-CODE_ID=$(echo $RES | jq -r '.logs[0].events[0].attributes[-1].value')
+TX=$(junod tx wasm store cw_erc20.wasm  --from <your-key> --chain-id=<chain-id> --gas auto --output json -y | jq -r '.txhash')
+CODE_ID=$(junod query tx $TX --output json | jq -r '.logs[0].events[-1].attributes[0].value')
```

**Comment by nullmames** [comment]
using -o json will not always result in the output being json. i have only ever had great success getting json output bybsetting the config.toml to output json. alternatively you can curl the json rpc.

**Comment by giansalex** [comment]
solved #23
